cmd/mattermost/commands/server_test.go

```diff
@@ -15,6 +15,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const (
+	UnitTestListeningPort = ":8066"
```

Comment:
Would we consider a random available port instead? I have bee known to use `:8066` to run a parallel server instance :)

Reply:
Fair point.  Detecting available port would be a bit more involved and probably not worth it, so I switched to a port very unlikely to be used.  Thanks.

Reply:
My apologies for an unsolicited comment. But if we set port to 0, kernel will assign a random port by itself, and then we can get query the network address to get the assigned port number if we want it in future.

Reply:
Done